apps/dashboard/app/views/apps/_app_group_rows.html.erb

```diff
@@ -0,0 +1,10 @@
+<%- group.apps.each do |app| %>
+  <%- app.links.each do |link| %>
```

Comment:
If role is "files" you could use https://getbootstrap.com/docs/3.3/components/#btn-dropdowns-split and put all of the app.links in the single <tr>, otherwise what you have above (one row for each app link).

Reply:
:+1: fixed.